src/libtls/tls_crypto.c

```diff
@@ -277,22 +277,24 @@ ENUM_NEXT(tls_signature_scheme_names,
 		  TLS_SIG_ECDSA_SHA1, TLS_SIG_ECDSA_SHA1, TLS_SIG_RSA_PKCS1_SHA1,
 	"ECDSA_SHA1");
 ENUM_NEXT(tls_signature_scheme_names,
-		  TLS_SIG_RSA_PKCS1_SHA256, TLS_SIG_RSA_PKCS1_SHA256, TLS_SIG_ECDSA_SHA1,
-	"RSA_PKCS1_SHA256");
+		  TLS_SIG_RSA_PKCS1_SHA224, TLS_SIG_ECDSA_SHA224, TLS_SIG_ECDSA_SHA1,
```

Comment:
Why do you think? Is it to clarify that not all of the added constants are usable/configurable?

I would suggest one comment just before `ENUM_BEGIN` rather than the same comment multiple times above all added signature constants. Do you agree?

```
/*
* Some peers, e.g. Firefox, send a wide range of signature algorithms. To prevent numeric identifiers in the log these algorithms are added here.
* 
* - RSA_PKCS1_SHA224
* - DSA_SHA224
* - ECDSA_SHA224
* - RSA_PKCS1_SHA256
* - DSA_SHA256
* - RSA_PKCS1_SHA384
* - DSA_SHA384
* - RSA_PKCS1_SHA512
* - DSA_SHA512
*/
```

Reply:
I don't think a comment is necessary. It would just help to see in the history why these old identifiers were added after such a long time.